controller/src/main/java/org/jboss/as/controller/ObjectTypeAttributeDefinition.java

```diff
@@ -195,9 +194,13 @@ public ModelNode resolveValue(ExpressionResolver resolver, ModelNode value) thro
     }
 
     protected void addValueTypeDescription(final ModelNode node, final String prefix, final ResourceBundle bundle,
```

Comment:
Since this was protected, please restore and deprecate the old variant, having it pass 'false' into this new one.